src/test/java/RandomGeneratorTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class RandomGeneratorTest {
+    @Test
+    @DisplayName("난수 값 6개 생성 테스트")
+    void 난수_값_6개_생성_테스트() {
+        List<Integer> numbers = RandomGenerator.generateUniqueRandomNumbers(6, 1, 45);
+        assertThat(numbers.size()).isEqualTo(6);
+    }
+
+    @Test
+    @DisplayName("값들의 범위 테스트")
+    void 값들의_범위_테스트() {
+        List<Integer> numbers = RandomGenerator.generateUniqueRandomNumbers(6, 1, 45);
+
+        for (int number : numbers) {
+            assertThat(number).isBetween(1, 45);
+        }
+    }
+
+    @Test
+    @DisplayName("값들의 중복 테스트")
```

Comment:
테스트 이름을 더 풀어서 적어보면 좋겠어요~
어떤상황에 무엇을 검증하는지요 ㅎㅎ

```suggestion
    @DisplayName("생성하는 난수들은 중복없이 고유한 값으로 생성된다")
```

Reply:
넵 확인했습니다!